src/quo2/components/navigation/floating_shell_button.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo2.components.navigation.floating-shell-button
+  (:require [quo.react-native :as rn]
+            [quo2.components.buttons.dynamic-button :as dynamic-button]))
+
+(defn dynamic-button-view [type dynamic-buttons style]
+  (when-let [{:keys [count on-press customization-color]} (get dynamic-buttons type)]
+    [dynamic-button/dynamic-button
+     {:type                type
+      :on-press            on-press
+      :count               count
+      :style               style
+      :customization-color customization-color}]))
+
+(defn floating-shell-button
+  "[floating-shell-button dynamic-buttons style]
+  dynamic-buttons
+  {:button-type {:on-press on-press :count count}}"
+  [dynamic-buttons style]
+  [rn/view {:style (merge {:flex-direction    :row
+                           :margin-horizontal 12} style)}
+   ;; Left Section
+   [rn/view {:style {:flex 1}}
+    [dynamic-button-view :search dynamic-buttons {:position :absolute
+                                                  :right    8}]]
+   ;; Mid Section (jump-to)
+   [dynamic-button-view :jump-to dynamic-buttons nil]
+   ;; Right Section   
+   [rn/view {:style {:flex 1}}
+    [rn/view {:style {:position       :absolute
+                      :flex-direction :row
+                      :right          0}}
```

Comment:
Multiple Views required:
- First view makes sure that left section and right section have equal width
- Second view makes sure that content is aligned in right side of right section

![image](https://user-images.githubusercontent.com/17097240/195475433-ce853da5-b970-4c25-ae0a-c1c676fb6863.png)